### PR TITLE
Fix default transport docs

### PR DESCRIPTION
**Pull request by msm1992**
## Purpose
Fixes Administer > Changing Default Transports doc.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md [hunk 1]
```diff
@@ -1,34 +1,29 @@
 # Changing the Default Transport
 
-On the back end, APIs are Apache Synapse configurations that WSO2 API Manager accesses through a transport. The default API Manager transport is the PassThrough transport, but you can configure a different default transport in your `axis2.xml` file. For example, to use the HTTP-NIO transport as the default, go to the `<APIM_HOME>/repository/conf/axis2` folder, open the `axis2.xml` file, and then in the "Transport Ins" and "Transport Outs" sections, comment out the PassThrough configurations and uncomment the configurations for the HTTP-NIO transport.
+On the back end, APIs are Apache Synapse configurations that WSO2 API Manager accesses through a transport. The default API Manager transport is the PassThrough transport, but you can configure a different default transport in your `deployment.toml` file.
```
Comment (Mariangela) [inline en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: Shall we reword this sentence [1] as [2].

[1] On the back end, APIs are Apache Synapse configurations that WSO2 API Manager accesses through a transport.
[2] "APIs are made up of Apache Synapse configurations that WSO2 API Manager accesses through a transport protocol."

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
 
 !!! info
-WSO2 products do not use the HTTP/S servlet transport configurations that are in `axis2.xml` file. Instead, they use Tomcat-level servlet transports, which are used by the management console in `<PRODUCT_HOME>/repository/conf/tomcat/catalina-server.xml` file .
+    WSO2 products do not use the HTTP/S servlet transport configurations that are in `axis2.xml` file. Instead, they use Tomcat-level servlet transports, which are used by the management console in `<PRODUCT_HOME>/repository/conf/tomcat/catalina-server.xml` file .
```
Comment (Mariangela) [inline en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: Shall we customize this as this doc is only for APIM.


[1]WSO2 products do not use the HTTP/S servlet transport configurations that are in `axis2.xml` file. Instead, they use Tomcat-level servlet transports, which are used by the management console in `<PRODUCT_HOME>/repository/conf/tomcat/catalina-server.xml` file .

[2] WSO2 API Manager do not use the HTTP/S servlet transport protocol configurations that are in the `axis2.xml` file. Instead, WSO2 API Manager's Management Console uses the Tomcat-level servlet transport protocols that are available in the `<API-M_HOME>/repository/conf/tomcat/catalina-server.xml` file.

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md [hunk 3]
```diff
@@ -1,34 +1,29 @@
...
--   [Connection throttling](#ChangingtheDefaultTransport-Connectionthrottling)
+-   [HTTP PassThrough Transport](#http-passthrough-transport)
+-   [Transport Receiver Parameters](#transport-receiver-parameters)
+-   [Transport Sender Parameters](#transport-sender-parameters)
+-   [Connection Throttling](#connection-throttling)
 
-### HTTP PassThrough transport
+### HTTP PassThrough Transport
 
 HTTP PassThrough Transport is the default, non-blocking HTTP transport implementation based on HTTP Core NIO and is specially designed for streaming messages. It is similar to the old message relay transport, but it does not care about the content type and simply streams all received messages through. It also has a simpler and cleaner model for forwarding messages back and forth. The two classes that implement the receiver and sender APIs are `org.apache.synapse.transport.passthru.PassThroughHttpListener` and `org.apache.synapse.transport.passthru.PassThroughHttpSender` , respectively. The PassThrough Transport does not require the binary relay builder and expanding formatter.
```
Comment (Mariangela) [inline en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: Shall we change [1] to [2].

[1] It is similar to the old message relay transport, but it does not care about the content type and simply streams all received messages through.

[2] It is similar to the old message relay transport, but it does not take the content type into consideration and simply streams all the messages that it receives.

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md [hunk 4]
```diff
@@ -147,13 +142,18 @@ The HTTP-NIO transport is a module of the Apache Synapse project. Apache Synapse
 </tbody>
 </table>
 
-### Connection throttling
+### Connection Throttling
+
+With the HTTP PassThrough, you can enable connection throttling to restrict the number of simultaneous open connections. To enable connection throttling, edit the `<PRODUCT_HOME>/repository/conf/deployment.toml` and add the following configuration under passthough_http configurations.
```
Comment (Mariangela) [inline en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: Shall we change [1] to [2].

[1] With the HTTP PassThrough, you can enable connection throttling to restrict the number of simultaneous open connections.

[2] Using the HTTP PassThroughtrasport protocol, you can enable connection throttling to restrict the number of simultaneous open connections.

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md (hunk 3 above)
Comment (Mariangela) [inline en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: `org.apache.synapse.transport.passthru.PassThroughHttpSender` ,

Remove space before , pls.

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md (hunk 4 above)
Comment (Mariangela) [inline en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: Shall we change [1] to [2]

[1] To enable connection throttling, edit the `<PRODUCT_HOME>/repository/conf/deployment.toml` and add the following configuration under passthough_http configurations.

[2] To enable connection throttling, edit the `<API-M_HOME>/repository/conf/deployment.toml` and add the following configuration under `passthough_http` configurations.

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md (hunk 1 above)
Comment (msm1992) [reply en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md (hunk 2 above)
Comment (msm1992) [reply en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md (hunk 3 above)
Comment (msm1992) [reply en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md (hunk 4 above)
Comment (msm1992) [reply en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md (hunk 3 above)
Comment (msm1992) [reply en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff

File: en/docs/Administer/ProductConfigurations/changing-the-default-transport.md (hunk 4 above)
Comment (msm1992) [reply en/docs/Administer/ProductConfigurations/changing-the-default-transport.md]: fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff

**Pull request merged**